ew/cmd/wep/wepcmds.py

```diff
@@ -1333,8 +1333,8 @@ async def divorce(cmd):
     elif user_data.poi != ewcfg.poi_id_dojo:
         response = "As much as it would be satisfying to just chuck your {} down an alley and be done with it, here in civilization we deal with things *maturely.* You’ll have to speak to the guy that got you into this mess in the first place, or at least the guy that allowed you to make the retarded decision in the first place. Luckily for you, they’re the same person, and he’s at the Dojo.".format(
             weapon.str_weapon)
-    elif user_data.life_state == ewcfg.life_state_juvenile:

```

Comment:
this was added to prevent a divorce bug iirc. since you need to enlist to use tools anyway i don't think this is necessary